app/Observers/GuardianObserver.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Observers;
+
+use App\Guardian;
+use App\User;
+
+class GuardianObserver
+{
+  /**
+   * Handle the guardian "creating" event.
+   *
+   * @param  \App\Guardian  $guardian
+   * @return void
+   */
+  public function creating(Guardian $guardian)
+  {
+    $this->setDefaultPassword($guardian);
```

Comment:
```suggestion
    $guardian->password = $guardian->createDefaultPassword(); 
```
i think lets just leave this as is without a method as its clear what we are doing